samples/sensor/bme680/prj.conf

```diff
@@ -0,0 +1,29 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_I2C_0=y
+CONFIG_SENSOR=y
+CONFIG_BME680=y
+
+## Additional configuration options
+# Alternate I2C address. Default is 0x76
+# Set the SDO pin to GND for 0x76 and VDDIO for 0x77
+#CONFIG_BME680_I2C_ADDR=0x77
+
+# Alternate I2C device. Default is I2C_0
+#CONFIG_BME680_I2C_MASTER_DEV_NAME=I2C_1
+
+# Alternate Temperature, Pressure and Humidity oversampling. Defaults are below
+# Options are 1X, 2X, 4X, 8X and 16X
+#CONFIG_BME680_TEMP_OVER_2X=y
+#CONFIG_BME680_PRESS_OVER_16X=y
+#CONFIG_BME680_HUMIDITY_OVER_1X=y
+
+# Alternate IIR Filter options. The IIR Filter is switched off by default.
+# Options are 2, 4, 8, 16, 32, 64, 128
+#CONFIG_BME680_FILTER_OFF=y
+
+# Alternate operation modes for the sensor. Default is LP or Low Power mode.
+# In the LP mode, fetch samples every 3 seconds.
+# In the ULP mode, fetch samples every 300 seconds.
+#CONFIG_BME680_HEATR_TEMP_ULP=y
+#CONFIG_BME680_HEATR_DUR_ULP=y
```

Comment:
Don't add all the commented out configs.

Reply:
How is it intended to know the possible configurations?